src/lib.rs

```diff
@@ -172,18 +173,17 @@ pub fn run_tests_generic(config: Config, file_filter: impl Fn(&Path) -> bool + S
     let target = config.target();
 
     // A channel for files to process
-    let (submit, receive) = crossbeam::channel::unbounded();
-
+    let (submit, receive) = unbounded();
     // Some statistics and failure reports.
     let failures = Mutex::new(vec![]);
     let succeeded = AtomicUsize::default();
     let ignored = AtomicUsize::default();
     let filtered = AtomicUsize::default();
 
-    crossbeam::scope(|s| -> Result<()> {
+    thread::scope(|s| -> Result<()> {
```

Comment:
CI is failing because `thread` is not in scope

```suggestion
    std::thread::scope(|s| -> Result<()> {
```

Reply:
Sorry, that's what I get for pushing in hurry and panic.
I amended the file!